src/index.tsx

```diff
@@ -6,18 +6,24 @@ import reportWebVitals from './reportWebVitals';
 import { Auth0Provider } from '@auth0/auth0-react';
 import { Provider } from 'react-redux';
 import { store } from './store/Store';
+import { loadState, saveState } from './store/LocalStorage'
 import { Front } from 'react-bootstrap-icons';
 
+store.subscribe(() => {
```

Comment:
Why are saving Redux Store to local storage?

Reply:
removed local storage, page refreshing was causing issues, but ended up fixing the issues, but did not remove local storage.